### ISSUE TITLE
Добавить подсистему валидации JSON по Open API

### ISSUE THREAD

**Issue by thvvmas** [firstBitSportivnaya/PSSL#158]
https://github.com/Segate-ekb/1c_OpenApi_validation

**Comment by thvvmas** [comment]
И вспомогательные функции к ней: инициализацию структуры объекта JSON из схемы и заполнение таблицы значений из массива JSON по схеме

**Comment by ivanmolodec** [comment]
А формирование спеки open api (и главное формирование веб-интерфейса) в отдельную ишью?

**Comment by thvvmas** [comment]
@ivanmolodec да, я вынесу в отдельную. Ее надо совмещать с подсистемой управления http-сервисами,чтобы дублирование не получилось
